src/elements/ngm-side-bar.ts

```diff
@@ -144,6 +145,15 @@ export class SideBar extends LitElementI18n {
         ${this.activePanel !== 'share' ? '' : html`
           <ngm-share-link></ngm-share-link>`}
       </div>
+      <div .hidden=${this.activePanel !== 'settings'} class="ngm-side-bar-panel">
+        <div class="ngm-panel-header">${i18next.t('lsb_settings')}
+          <div class="ngm-close-icon" @click=${() => this.activePanel = ''}></div>
+        </div>
+        <div class="language-settings">
+          <label>${i18next.t('lsb_langs')}</label>
+          <div id="langs" class="ui horizontal selection list"></div>
```

Comment:
Something like this should work:
```suggestion
          <div id="langs" class="ui horizontal selection list">
          ${SUPPORTED_LANGUAGES.map(lang => html`
            <div class="item lang-${lang}" @click="${() => i18next.changeLanguage(lang)}">${lang.toUpperCase()}</div>
           }
  `);
          </div>
```

Reply:
yes, done in https://github.com/swissgeol/ngm/pull/667/commits/6f51e81fa90278d807f3623d6175286e9729f4ac